app/services/lesson_previews_remover.rb

```diff
@@ -0,0 +1,11 @@
+class LessonPreviewsRemover
```

Comment:
Appologies @JuanVqz, I think we got our wires crossed with whats needed here. We need a job instead of a service. Something very similar to what we have for discarding projects https://github.com/TheOdinProject/theodinproject/blob/main/app/jobs/discard_project_submission_job.rb 

```ruby
class RemoveExpiredPreviewsJob
  include Sidekiq::Worker
  sidekiq_options retry: 1, dead: false

  def perform
    LessonPreview.expired.each(&:destroy)
  end
end
```

Then we'll need a rake task to trigger the job everyday using Cron on Heroku: https://github.com/TheOdinProject/theodinproject/blob/main/lib/tasks/project_submissions.rake

Reply:
I wanted to use that service in the job. but okay, I'll put it all together in the job's body, :+1: